github/actions_oidc.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// OIDCSubjectClaimCustomizationTemplate represents an OIDC subject claim customization template
+type OIDCSubjectClaimCustomizationTemplate struct {
+	UseDefault       *bool    `json:"use_default,omitempty"`
+	IncludeClaimKeys []string `json:"include_claim_keys"`
+}
+
+// GetOrganizationOIDCSubjectClaimCustomizationTemplate gets the subject claim customization template for an organization.
+//
+// GitHub API docs: https://docs.github.com/en/rest/actions/oidc#get-the-customization-template-for-an-oidc-subject-claim-for-an-organization
+func (s *ActionsService) GetOrganizationOIDCSubjectClaimCustomizationTemplate(ctx context.Context, org string) (*OIDCSubjectClaimCustomizationTemplate, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/actions/oidc/customization/sub", org)
+	return s.getOIDCSubjectClaimCustomizationTemplate(ctx, u)
+}
+
+// GetRepositoryOIDCSubjectClaimCustomizationTemplate gets the subject claim customization template for a repository.
+//
+// GitHub API docs: https://docs.github.com/en/rest/actions/oidc#get-the-customization-template-for-an-oidc-subject-claim-for-a-repository
+func (s *ActionsService) GetRepositoryOIDCSubjectClaimCustomizationTemplate(ctx context.Context, owner, repo string) (*OIDCSubjectClaimCustomizationTemplate, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/actions/oidc/customization/sub", owner, repo)
+	return s.getOIDCSubjectClaimCustomizationTemplate(ctx, u)
+}
+
+func (s *ActionsService) getOIDCSubjectClaimCustomizationTemplate(ctx context.Context, url string) (*OIDCSubjectClaimCustomizationTemplate, *Response, error) {
+	req, err := s.client.NewRequest("GET", url, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	oidcSubjectClaimCustomizationTemplate := new(OIDCSubjectClaimCustomizationTemplate)
```

Comment:
Let's please make this temporary variable just a bit shorter. Maybe something like:

```suggestion
	tmpl := new(OIDCSubjectClaimCustomizationTemplate)
```